### ISSUE TITLE
sir am using GTPA010 GPS module with arduino uno am getting character and err but not getting any lat or long

### ISSUE THREAD

**Issue by Punithpeshva** [mikalhart/TinyGPS#4]
am using GTPAO10 GPS module with antenna i used your library it will show  some values for char and err but not showing any lat or long  just shows garbage characters pls help me out

**Comment by mikalhart** [comment]
Probably a baud rate mismatch?  I think GTPAO10 defaults to 9600, but the sample code is 4800.

**Comment by Punithpeshva** [comment]
I changed it to 9600 but end up with same result :(

**Comment by mikalhart** [comment]
Please try to Serial.write() each character coming from the GPS, just to make sure you are getting readable data.

**Comment by Punithpeshva** [comment]
Sorry I didn't get  using serial.write where the data will be written I
have used serialprint bt not write sorry that's why asked and thank you so
much for your concern and reply
And one doubt if device is not getting readable data means what's the fault
may be??

**Comment by mikalhart** [comment]
Make a test loop that just does this:

void loop()

{

   if (ss.available() > 0)

```
  Serial.write(ss.read());
```

}

Assuming you have all the baud rates correctly set up for ss and Serial, this should print readable characters on the serial monitor.

**Comment by Punithpeshva** [comment]
Yeah its giving some characters bt not showing exact co-ordinates its
showing wired data help out sir

**Comment by Punithpeshva** [comment]
Sir sorry for disturbing do i have any solutions sir??am ready to change my
gps device,which i have to buy to work with your code??

**Comment by Punithpeshva** [comment]
when i use serial.Write its just shwoing "9999"
